_posts/en/newsletters/2020-04-29-newsletter.md

```diff
@@ -0,0 +1,330 @@
+    continue indefinitely, so this new problem gives LN developers an
+    additional reason to prioritize the implementation of improved
+    onchain fee management.  In the interim, node operators concerned
+    about the attack may wish to increase their [cltv_expiry_delta][]
+    to give preimage settlement transactions more time to confirm.
+    Current defaults in popular LN nodes are [14][cl ced] for
+    C-Lightning, [40][lnd ced] for LND, [72][rl ced] for Rust-Lightning,
+    and [144][eclair ced] for Eclair.  Note that increasing the value
+    will make your channels less desirable to spenders,
+    as higher values increases the normal worst case
+    amount of time a payment could be stuck waiting to be settled.
+
+- **Multiparty vault architecture:** Antoine "Darosior" Poinsot
+  [announced][darosior revault] a demo implementation of a vaults
+  [covenant][topic covenants] prototype based on the same basic
+  presigned transaction concept mentioned in [last week's
+  newsletter][news94 bishop vault].  This new implementation, named
+  *Revault*, focuses on storing funds shared between multiple parties
+  with multisig security.  The protocol allows a subset of the parties
+  to initiate a withdrawal process by getting a beacon transaction
+  confirmed; if the other parties to the vault object to the withdrawal,
+  they have the opportunity to broadcast a second transaction that
+  returns the funds to an emergency address in the vault.  If there's no
+  objection within a certain amount of time, another transaction can
+  complete the withdrawal of the funds.  Poinsot is seeking feedback on
+  the proposal.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What are the potential attacks against ECDSA that would be possible if we used raw public keys as addresses?]({{bse}}95123)
+  Pieter Wuille answers by summarizing the argument for using public key hashes
+  over public keys in addresses, namely, that it slows an attacker with quantum
+  computing capabilities. He goes on to list reasons why that purported argument
+  could be overstated and give a false sense of security.
+
+- [What is meant by DEFAULT_ANCESTOR_LIMIT in child pays for parent?]({{bse}}95473)
+  User anu asks about Bitcoin Core's [DEFAULT_ANCESTOR_LIMIT][bitcoin core default ancestor limit]
+  regarding the [Child-Pays-For-Parent (CPFP)][topic cpfp] fee bumping
+  technique. Murch goes on to explain that this default policy helps prevent spam
```

Comment:
nit: "goes on to" doesn't seem quite right here without Murch having done something earlier. I think just "Murch explains" is better.